paper.bib

```diff
@@ -0,0 +1,80 @@
+@misc{HOMER,
```

Comment:
Do we want to add the zenodo release?

Reply:
Good idea - can you add this please? Do you think we should link it to [10.5281/zenodo.6925534](https://doi.org/10.5281/zenodo.6925534) or a specific previous version?

Reply:
I've included a reference to the release above, but let me know if you think a different one would be more appropriate

Reply:
No, I think that should be fine